ProjectCoimbra.UWP/DataAccessLibrary/DataAccess.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Licensed under the MIT License.
+
+namespace DataAccessLibrary
+{
+    using Microsoft.Data.Sqlite;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Windows.Storage;
+
+    /// <summary>
+    /// Data access class for the local User table in SQLite database
+    /// </summary>
+    public static class DataAccess
+    {
+        /// <summary>
+        /// Initializes the database, which is called when application starts
+        /// </summary>
+        public async static void InitializeDatabase()
+        {
+            await ApplicationData.Current.LocalFolder.CreateFileAsync("coimbra.db", CreationCollisionOption.OpenIfExists);
```

Comment:
Should we make "coimbra.db" a Static Read only String?

Reply:
or even better, have **dbPath** to be a static string

Reply:
I don't know if a static path would work (maybe there's a url handler). It would be good if we move strings to be static, just to avoid magic values, however the speed impact i negligible, I think, as the string is usually used only once vs server applications.

Reply:
Makes sense, seems to work for me anyways when using static